.gitignore

```diff
@@ -37,4 +37,5 @@ Theano.suo
 .ipynb_checkpoints
 .pydevproject
 .ropeproject
-core
\ No newline at end of file
+core
+.idea
```

Comment:
This is generated by the Intellij IDEA Python plugin, and probably PyCharm does the same.